jobs/scripts/nightly-samba-builds/nightly-samba-builds.sh

```diff
@@ -60,7 +62,12 @@ then
 	yum -y install "https://releases.hashicorp.com/vagrant/${VAGRANT_VERSION}/vagrant_${VAGRANT_VERSION}_x86_64.rpm"
 fi
 
-vagrant plugin install vagrant-libvirt
+# Since CentOS 8 is based on Fedora 32, following configuration is required for
+# libvirt plugin installation when upstream rpm is consumed.
+# See bullet point "Fedora 32 + upstream Vagrant:" from https://github.com/vagrant-libvirt/vagrant-libvirt#installation
+[ ${CENTOS_VERSION} = "8" ] && VAGRANT_CONFIG_PARAMS="with-libvirt-include=/usr/include/libvirt with-libvirt-lib=/usr/lib64" || true
+
+CONFIGURE_ARGS=\"${VAGRANT_CONFIG_PARAMS}\" vagrant plugin install vagrant-libvirt
```

Comment:
Is escaping the `"`'s really needed?

Reply:
Hm..there are two configuration options defined separated with [space].

We could either escape [space] directly within `VAGRANT_CONFIG_PARAMS` or use `export CONFIGURE_ARGS="with-libvirt-include=/usr/include/libvirt with-libvirt-lib=/usr/lib64"` to avoid escaping `"`s.

Reply:
I think it works fine if you use
```shell
CONFIGURE_ARGS="${VAGRANT_CONFIG_PARAMS}" vagrant plugin install vagrant-libvirt
```

Reply:
My bad. You are right.